integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -371,6 +374,107 @@ def test_hash_reduction_pivot_without_nans(data_gen, conf):
             .agg(f.sum('c')),
         conf=conf)
 
+_repeat_agg_column_for_collect_op = [
```

Comment:
Why "has_aggregate"? In cudf, collect list/set are only sort_aggregages.

Reply:
@ttnghia This test is not named after how cudf implements something.  What is more `cudf` hides that detail from users so any test outside of cudf itself That is a hidden implementation detail. It is named after how Spark implemented it, and we are trying to get coverage on that operator.

That said CollectList and CollectSet apparently are showing up as `ObjectHashAggregateExec` and there are also some `SortAggregateExec` tests in here too. At some point we should either split this up into separate files or rename it.